codelabs/codelab-03/README.md

```diff
@@ -17,118 +17,119 @@ Before starting this codelab, run `git pull` in the `389Lspring18` directory to
 
 ### Tutorial
 
-**It's a good idea to read/skim the entire codelab first so you have an idea of what you're doing. At a minimum, look at the submission section to _avoid having to redo your work_ to obtain screenshots**
+**It's a good idea to read/skim the entire codelab first so you have an idea of what you're doing. Keep in mind that you will be submitting screenshots of some command outputs, so make sure to read the submission section (at a minimum) located near the bottom to understand what you will be turning in**
 
 #### Back to buckets
 
-Fist, download the image `canyon.jpg`
-![here](../../../media/codelabs/codelab-03/canyon.jpg)
+Fist, download the image ![canyon.jpg](../../../media/codelabs/codelab-03/canyon.jpg)
```

Comment:
I was thinking you could keep both links (also Fist -> First):

First, download the image [`canyon.jpg`] (../../../media/codelabs/codelab-03/canyon.jpg)

![canyon.jpg] (../../../media/codelabs/codelab-03/canyon.jpg)